source/docs/software/python/index.rst

```diff
@@ -0,0 +1,11 @@
+FRC Python Programming
+======================
+
+.. toctree::
+   :maxdepth: 2
+   :titlesonly:
+
+   /docs/zero-to-robot/step-2/python-setup
+   /docs/zero-to-robot/step-2/frc-game-tools
+   pyproject_toml
+   subcommands/index
```

Comment:
Is this how you wanted it formatted?
![image](https://github.com/wpilibsuite/frc-docs/assets/4885091/68f20535-ef03-4462-b81a-591d7b899389)

Reply:
My thought was that in the future someone would document some of the other subcommands